code/obj/item/storage/bible.dm

```diff
@@ -39,15 +39,24 @@ var/global/list/bible_contents = list()
 			JOB_XP(user, "Chaplain", 2)
 		else
 			var/mob/living/H = M
-			if( istype(H) )
-				if( prob(25) )
-					H.delStatus("bloodcurse")
-					H.cure_disease_by_path(/datum/ailment/disease/cluwneing_around/cluwne)
-				if(prob(25))
-					H.cure_disease_by_path(/datum/ailment/disability/clumsy/cluwne)
 			M.HealDamage("All", heal_amt, heal_amt)
 			if(prob(40))
 				JOB_XP(user, "Chaplain", 1)
+			if( !istype(H) )
+				return
+			if( prob(25) )
+				if (iscluwne(H))
+					H.cure_disease_by_path(/datum/ailment/disease/cluwneing_around)
+					H.cure_disease_by_path(/datum/ailment/disease/cluwneing_around/cluwne)
+					for(var/obj/item/clothing/W in H) //The Chaplain is able to cure the disease AND the curse. Science can only deal with the disease
+						if (W.cant_self_remove && W.cant_other_remove)//this might not be a great way to do this.
```

Comment:
Yeah - wouldn't this remove any electropacks or whatever

Reply:
know it's not your code

Reply:
Electropacks wouldn't have been affected, but you're right it could have been used to eg: strip the Slasher. I've updated it to also check for "cursed": at the moment that combination of terms only corresponds to cluwne gear.